cockatrice/src/abstractcarddragitem.cpp

```diff
@@ -1,41 +1,52 @@
 #include "abstractcarddragitem.h"
 #include "carddatabase.h"
 #include <QCursor>
-#include <QGraphicsSceneMouseEvent>
 #include <QDebug>
+#include <QGraphicsSceneMouseEvent>
 #include <QPainter>
 
 static const float CARD_WIDTH_HALF = CARD_WIDTH / 2;
 static const float CARD_HEIGHT_HALF = CARD_HEIGHT / 2;
 const QColor GHOST_MASK = QColor(255, 255, 255, 50);
 
-AbstractCardDragItem::AbstractCardDragItem(AbstractCardItem *_item, const QPointF &_hotSpot, AbstractCardDragItem *parentDrag)
+AbstractCardDragItem::AbstractCardDragItem(AbstractCardItem *_item,
+                                           const QPointF &_hotSpot,
+                                           AbstractCardDragItem *parentDrag)
     : QGraphicsItem(), item(_item), hotSpot(_hotSpot)
 {
-    if (parentDrag) {
+    if(parentDrag)
+    {
         parentDrag->addChildDrag(this);
         setZValue(2000000007 + hotSpot.x() * 1000000 + hotSpot.y() * 1000 + 1000);
-    } else {
-        if ((hotSpot.x() < 0) || (hotSpot.y() < 0)) {
+    }
+    else
+    {
+        if((hotSpot.x() < 0) || (hotSpot.y() < 0))
+        {
             qDebug() << "CardDragItem: coordinate overflow: x =" << hotSpot.x() << ", y =" << hotSpot.y();
             hotSpot = QPointF();
-        } else if ((hotSpot.x() > CARD_WIDTH) || (hotSpot.y() > CARD_HEIGHT)) {
+        }
+        else if((hotSpot.x() > CARD_WIDTH) || (hotSpot.y() > CARD_HEIGHT))
+        {
             qDebug() << "CardDragItem: coordinate overflow: x =" << hotSpot.x() << ", y =" << hotSpot.y();
             hotSpot = QPointF(CARD_WIDTH, CARD_HEIGHT);
         }
         setCursor(Qt::ClosedHandCursor);
         setZValue(2000000007);
     }
-    if (item->getTapped())
-        setTransform(QTransform().translate(CARD_WIDTH_HALF, CARD_HEIGHT_HALF).rotate(90).translate(-CARD_WIDTH_HALF, -CARD_HEIGHT_HALF));
+    if(item->getTapped())
```

Comment:
I’d like to require brackets after any conditional whether they’re one line or not

Reply:
Yes, we should do that

Reply:
Afaik unfortunately no such option exists in clang-format.